raiden/storage/utils.py

```diff
@@ -47,10 +47,28 @@ def __getattr__(self, item):
 );
 """
 
+DB_CREATE_MATRIX_USER_IDS = """
+CREATE TABLE IF NOT EXISTS matrix_user_ids (
+    address_identifier INTEGER PRIMARY KEY,
+    address BINARY NOT NULL,
+    userids TEXT NOT NULL,
```

Comment:
Optional: If you want to fully normalize this there should be a `matrix_userids` table as well which is referenced from this.